Makefile

```diff
@@ -107,6 +107,34 @@ install:
 
 # Browser-specific hosts targets
 
+.PHONY: hosts-iridium
+hosts-iridium:
```

Comment:
Please also update README, add Iridium to the bottom of the tables in [Configure browsers](https://github.com/browserpass/browserpass-native#configure-browsers) section